packages/react-components/react-avatar/src/components/AvatarGroupItem/useAvatarGroupItemStyles.ts

```diff
@@ -59,83 +59,104 @@ const useOverflowLabelStyles = makeStyles({
  * Styles for the stack layout
  */
 const useStackStyles = makeStyles({
-  base: {
+  overflowButton: {
+    ...shorthands.outline('1px', 'solid', tokens.colorBrandBackground2),
     '&::after': {
       content: "''",
       position: 'absolute',
       display: 'inline-flex',
       // Border is used instead of outline due to a bug in webkit browsers where border-radius is ignored in outline.
       ...shorthands.borderColor(tokens.colorNeutralBackground2),
+      // ...shorthands.borderColor(tokens.colorNeutralBackground2),
       ...shorthands.borderRadius(tokens.borderRadiusCircular),
       ...shorthands.borderStyle('solid'),
+      width: '100%',
+      height: '100%',
 
       '@media (forced-colors: active)': {
         forcedColorAdjust: 'none',
       },
     },
-  },
-  overflowButton: {
-    '&:focus::after': {
+
+    '&:focus::before': {
       ...shorthands.borderColor('transparent'),
     },
   },
-  thick: {
-    '&::after': {
+  nonOverflowButton: {
+    '&::before': {
+      content: "''",
+      position: 'absolute',
+      display: 'inline-flex',
+      backgroundColor: tokens.colorNeutralBackground2,
+      // Border is used instead of outline due to a bug in webkit browsers where border-radius is ignored in outline.
+      ...shorthands.borderColor(tokens.colorNeutralBackground2),
+      // ...shorthands.borderColor(tokens.colorNeutralBackground2),
```

Comment:
_nit:_ Remove this if it's unused

```suggestion
```

Reply:
Sorry, I'm still working on it. I just needed to test Screener and it doesn't run in a draft PR.